main/ertp/api/issuer.md

```diff
@@ -1,214 +1,263 @@
+instead of making the strings yourself.
+- `MathKind.NAT` (`nat`): Used with fungible assets. `amount` values are natural numbers (non-negative integers). Default value.
+- `MathKind.STRING_SET` (`strSet`): Used with non-fungible assets. `amount` values are strings.
+- `MathKind.SET` (`set`): Used with non-fungible assets. `amount` values are objects or records with multiple properties.
 
-## makeIssuerKit(allegedName, mathHelperName)
-- `allegedName` `{String}`
-- `mathHelpersName` `{String}`
-- Returns: `{ mint, issuer, amountMath, brand }`
-
-Makes an issuer and related objects. 
-
-The `allegedName` is available from the brand in asset descriptions. The `allegedName` is useful for debugging and double-checking assumptions, but
-should not be trusted.
-
-The `mathHelpersName` will be used to import a specific mathHelpers from the mathHelpers library. For example, `natMathHelpers`, the default, supports
-natural numbers and is used for basic fungible tokens.
+```js
+import { MathKind, makeIssuerKit } from '@agoric/ertp';
+makeIssuerKit('quatloos`); // Defaults to 'MathKind.NAT'
+makeIssuerKit('foobars', 'MathKind.STRSET');
+makeIssuerKit('kitties', MathKind.SET');
+```
 
 ```js
-const { issuer, mint, amountMath } = makeIssuerKit('bucks');
+const { issuer: quatloosIssuer, mint: quatloosMint, amountMath: quatloosAmountMath } = makeIssuerKit('quatloos');
 // This is merely an amount, describing assets.
-const bucks2 = amountMath.make(2);
+const quatloos2 = quatloosAmountMath.make(2);
 
-const { mint, issuer, amountMath } = makeIssuerKit('alamedaCountyPropertyTitle', 'strSet');
+const { mint: titleMint, issuer: titleIssuer, amountMath: titleAmountMath } = makeIssuerKit('alamedaCountyPropertyTitle', 'strSet');
 
 // These are merely amounts describing digital assets, not minting assets.
-const cornerProperty = amountMath.make(harden['1292826']);
-const adjacentProperty = amountMath.make(harden['1028393']);
-const combinedProperty = amountMath.make(harden['1292826', '1028393']);
+const cornerProperty = titleAmountMath.make(harden['1292826']);
+const adjacentProperty = titleAmountMath.make(harden['1028393']);
+const combinedProperty = titleAmountMath.make(harden['1292826', '1028393']);
 ```
 
-## issuer.getBrand()
-- Returns: `{Brand}` - The brand for the issuer.
+## issuer.getAllegedName()
+- Returns: `{allegedName}`
+
+Returns the `allegedName` for this `issuer`.
 
-Get the Brand for this Issuer. The Brand indicates the kind of digital asset and is shared by the mint, the issuer, and any purses and payments of this particular kind. The brand is not closely held, so this function should not be trusted to identify an issuer alone. Fake digital assets and amounts can use another issuer's brand.
+An alleged name is a human-readable string name 
+of a kind of digital asset. The alleged name 
+should not be trusted as accurate, since it 
+is not provided by a trusted source.
```

Comment:
I commented elsewhere about this reason not being quite right. The real reason is that there can be multiple issuers/mints/brands with the same alleged name since there is no public registry or expectation of uniqueness. That means that the name itself can't be used to correctly identify an issuer. It's the brand that does that. The name is just the human readable version that is helpful for debugging.

Reply:
The issuer is a very trusted source, which is part of why this is wrong. It's just that I can create an issuer with the allegedName 'Bitcoin' and it doesn't make it THE bitcoin issuer.